src/Mailer/Email.php

```diff
@@ -1925,6 +1929,25 @@ protected function _renderTemplates($content)
             $rendered[$type] = rtrim($rendered[$type], "\n");
         }
 
+        /* Embed images inline in html templates */
+        if (!empty($rendered['html'])) {
+            $rendered['html'] = str_replace(['file:', 'file://', 'cid://'], 'cid:', $rendered['html']);
```

Comment:
Of course. Without this line (1934) the url format for embed images is "cid:/absolute/path/for/image.jpg"